distributed/profile.py

```diff
@@ -355,3 +355,50 @@ def plot_figure(data, **kwargs):
     fig.grid.visible = False
 
     return fig, source
+
+
+def _remove_py_stack(frames):
+    for entry in frames:
+        if entry.is_python:
+            break
+        yield entry
+
+
+def llprocess(frames, child, state):
+    if not frames:
+        return
+    frame = frames.pop()
+    if frames:
+        state = llprocess(frames, frame, state)
+
+    addr = hex(frame.addr - frame.offset)
+    ident = ';'.join(map(str, (frame.name, '<c>', addr)))
```

Comment:
Is "<c>" the right name here?  Are we likely to get information from other compiled languages like Fortran or C++?  If so, what is a good alternative here?  "<compiled>" ?  "<low-level>" ?

Reply:
I don't understand, where do you see ""?

Reply:
Ah, this is because I was using text like `<foo>` which github chose wisely to hide.  I've now wrapped these in backticks above.  Hopefully now my comment from before makes more sense.

Reply:
Thanks for clarifying @mrocklin. Your question is indeed a good one, I can't answer about Fortran, but from C++ symbols we definitely will. I'm fine using `<low-level>`instead, but `<compiled>` feels confusing somehow.

Reply:
`<low-level>` it is then!  (unless someone can think of something better)